internal/cmd/run.go

```diff
@@ -172,9 +173,11 @@ func runCmd() *cobra.Command {
 				return err
 			}
 
-			err = promptEnvVars(cmd, sessionEnvs, runBlocks...)
-			if err != nil {
-				return err
+			if isTerminal(os.Stdout.Fd()) && !skipPromptVars {
```

Comment:
Wondering if `skipPromptVars` should be set to true if it's not a terminal and the flag's not changed explicitly. That way one could always enforce behavior with `--skip-prompts=true` or `--skip-prompts=false`.

Reply:
What about ☝️ @duvanmonsa ?

Reply:
I changed the default value to `true` @sourishkrout